grammar.js

```diff
@@ -1381,6 +1384,47 @@ module.exports = grammar({
       optional($.returning),
     ),
 
+    _upsert_statement: $=> seq(
```

Comment:
I don't think any dialect actually uses a keyword `upsert` but imo it'd still be clearer to call this `_merge_statement`